receipt_for_lovely_loveseats/receipt.py

```diff
@@ -0,0 +1,24 @@
+class Receipt:
+  def __init__(self, products = []):
+    self.products = products
+
+  def receipt_body(self, items = [], totals = {}):
+    receipt_line_items = []
+
+    receipt_line_items.append('Customer Items:')
+    receipt_line_items.extend(self.__receipt_product_line_item_list(items))
+    receipt_line_items.append('')
+    receipt_line_items.append(f'Subtotal: ${self.__format_number(totals["subtotal"])}')
+    receipt_line_items.append(f'Sales tax ({self.__format_number(totals["sales_tax_rate"] * 100)}%): ${self.__format_number(totals["sales_tax_total"])}')
+    receipt_line_items.append(f'Total: ${self.__format_number(totals["total"])}')
```

Comment:
This can be cleaned up by transforming the values in `totals` outside of this. It'll make the code easier to read.

Reply:
I also wish there was a nice way to deconstruct...